vue/src/Version/VersionEdit.vue

```diff
@@ -312,8 +312,10 @@ export default defineComponent({
 
         if (this.canPublishToLive) {
           this.version.environments = ['live'];
-        } else if (!this.version.environments) {
```

Comment:
@snake14 Why did we remove this check, not able to understand

Reply:
@AltamashShaikh I noticed that `this.version` is defined on line 306 and doesn't include the `environments` property, so `this.version.environments` is always going to be undefined and doesn't need to be checked.

Reply:
understood :+1: